### ISSUE TITLE
Running Evaluation for KITTI Test

### ISSUE THREAD

**Issue by nbansal90** [zacjiang/GMA#20]
Hey @zacjiang ,

Thank you for sharing your work! 
@zacjiang, I was looking to evaluate the pre-trained model on the KITTI test set. I have completed the repository set up, and got it running according to instructions mentioned on GitHub. I was able to reproduce the results mentioned in the paper in Table 2 for the KITTI train dataset. 

But When I run it for the KITTI test, the execution of evaluation.py fails. This is because in evaluation.py file, It expects 4 outputs from the data loader, https://github.com/zacjiang/GMA/blob/2f1fd29468a86a354d44dd25d107930b3f175043/evaluate.py#L348-L355.  Whereas if the split is test then according to the dataset file here:
https://github.com/zacjiang/GMA/blob/2f1fd29468a86a354d44dd25d107930b3f175043/core/datasets.py#L38-L46. It returns only three values. 

Then in that case how do i get numbers for KITTI Test split.

Regards,
Nitin Bansal

**Comment by Fanstandy** [comment]
The same problem exists with the Sintel test set.
